admin-policy.md

```diff
@@ -0,0 +1,75 @@
+- It is *highly* recommended that admins idle on ##hx-admin so that the community can easily have a short discussion or ask questions directly to the admins in a public area, without polluting the main HX group. HX Meta is also alright for this purpose, but an IRC conversation can often provide immediate feedback.
+
+## Election of Admins
+
+- The current admin team will determine how many admins are elected for the next term. 
+- 14 days before the end of a term, the web interface will allow candidates to mark themselves as running. 
+  - Candidates must be nominated by three group members via the web interface (after they have marked themselves as running) to be put on the ballot. Each group member can nominate one member per term.
+- 7 days before the end of a term, the web interface will allow group members to vote for a candidate. 
+- Votes will stop being accepted after 11:59PM GMT on the day the term ends. The current admins will use the voting result to add/remove themselves, if needed.
+- The voting system will publish a freely downloadable list of UUIDs and their vote after votes stop being accepted.
+
+### Voting
+
+- Votes are counted via a voting web interface.
+  - In the event the web interface becomes inaccessible while voting is in progress, the voting period and moderation term may be extended to the nearest hour based on the outage length.
+- The group will have one official maintainer for the voting web interface.
+  - The maintainer can be removed and replaced via an unanimous vote among the current elected admins.
+  - The maintainer is responsible for producing the results of the election for each term, ensuring the voting application is secure, and maintaining its reliability. 
+    - The maintainer may remove clearly spam candidates, and the current admins may veto any removals.
+  - The maintainer must not run in any election they maintain.
+- The voting system will not allow non-group members to vote; it may additionally deny some users the ability to vote if the user's account appears fraudulent.
+- The voting system will securely generate a v4 UUID for each user after they cast their first vote. When the user votes, their UUID will be stored along with their vote. When the election concludes, the UUID and vote will be published to allow auditing of the results.
+- Under this model (to be improved in the future), the maintainer is able to access the database linking votes to people, though the integrity of the results is relatively sound.
+
+## Removal of Admins
+
+Administrators may be removed by an unanimous vote among the other current admins. The current administrators may choose to hold a temporary election or wait until the next election if an administrator is voted out. 
+
+## Purpose of Administration
+
+- Lock/close posts (and remove when content policy specifies).
+- Ban users and deal with ban appeals (see the ban policy for more details).
+- Manage the group name/cover photo and its promotion.
```

Comment:
And adding new users that have accumulated in the Add Request queue. :-P I feel like that's what I do most of the time.